Sources/HipMobileUI/App.xaml

```diff
@@ -22,10 +22,10 @@
              xmlns:converters1="clr-namespace:PaderbornUniversity.SILab.Hip.Mobile.UI.Converters;assembly=HipMobileUI">
     <Application.Resources>
         <ResourceDictionary>
-            <Color x:Key="PrimaryColor">#3F51B5</Color>
-            <Color x:Key="PrimaryDarkColor">#303F9F</Color>
-            <Color x:Key="AccentColor">#FF9800</Color>
-            <Color x:Key="AccentDarkColor">#E65100</Color>
+            <Color x:Key="PrimaryColor">#7FACFF</Color>
+            <Color x:Key="PrimaryDarkColor">#0149D1</Color>
+            <Color x:Key="AccentColor">#FFE57F</Color>
+            <Color x:Key="AccentDarkColor">#FFCC00</Color>
```

Comment:
The two red colors (#F22F0C and #FF937F) from the **Corporate Design Manual** should also be included. You could rename the _AccentColor_ to _SecondaryColor_ and use _TertiaryColor_ for the red ones.

Reply:
ok

Reply:
Do you want to keep the name _AccentColor_? I think it would be more fitting to call this SecondaryColor since it is no longer only an accenting color but one of our 3 main colors.